src/util/dwarf.rs

```diff
@@ -1335,7 +1436,11 @@ pub fn subroutine_def_string(
                 continue;
             }
             let variable = process_variable_tag(info, tag)?;
-            writeln!(out, "    // -> {}", variable_string(info, typedefs, &variable, false)?)?;
+            writeln!(
+                out,
+                "    // -> {}",
+                variable_string(info, typedefs, &variable, false)?
+            )?;
```

Comment:
I tried `cargo fmt` but it seemed to make lots of useless change like this (and for lots of other files too). Is there some special config I need?

Reply:
You’ll want to use `cargo +nightly fmt`, since I’m using nightly-only rustfmt settings. (I should put this in the README)